### ISSUE TITLE
cheat engine wont open i dont have badlion client installed. the tutorial will open but nothing else in the cheat engine folder.

### ISSUE THREAD

**Issue by furret2** [cheat-engine/cheat-engine#1377]

**Comment by cheat-engine** [comment]
do a `driverquery /v > bla.txt` and upload bla.txt here so we can see what else might be blocking you

**Comment by FaelRocker** [comment]
I have the same problem as him. Never installed this badlion something.
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/5418141/bla.txt)

**Comment by cheat-engine** [comment]
So the tutorial runs but cheat engine doesn't? Because that implies the installer does work properly.

**Comment by furret2** [comment]
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/5419079/bla.txt)
my bla.txt

**Comment by fade2gray** [comment]
@cheat-engine 

> ... the other common reason for failure is a adblocker on the network ...

Would that include running the likes of a [Pi-Hole](https://pi-hole.net/)?

**Comment by cheat-engine** [comment]
yes, that seems to block ce's installer

**Comment by FaelRocker** [comment]
> 
> 
> So the tutorial runs but cheat engine doesn't? Because that implies the installer does work properly, and the other common reason for failure is a adblocker on the network which sends data that crashes the installer

Yes, the tutorial runs just fine. The adblockers that I use are the browser plugins type.

**Comment by cheat-engine** [comment]
I don't know.  It could be Nox,  xscanner, mi2c, cbproc, etc...  (You have many 'custom' drivers in your system)

**Comment by Lennart23232** [comment]
i have the same problem 
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/5461934/bla.txt)
